README.md

```diff
@@ -0,0 +1,17 @@
+# BeGUI
+BeGUI is a simple to use programmer's tool which allows for rapid development
+of a program.
+
+![BeGUI prefs screenshot](begui.jpg)
+
+
+Building
+--------
+
+Simply go into BeGUI directory and type ``bash creationx86.script``.
+
+
+Copyright
+------------
+
+Original 1999 version by Ed Musgrove, enhanced and updated for Haiku in 2018 by Arnav Bhatt.
```

Comment:
Same here, drop the enhanced by comment.  If/when you make significant changes and improments to the program it might be appropriate to add a comment like like, but at this stage we are just doing simple maintenance.  The original author still did the majority of the work here.  Your contributions will be noted in your commit messages etc.